### PR TITLE
feat:homepage

### LINKED ISSUE #8: Front page asking to choose one of the given datasets
a simple webpage which shows the available datasets and send a get request about the choice.

### PR DESCRIPTION
**Pull request by AkMo3**
closes #8

### CODE REVIEW

**Comment by RoneelV** [comment]
Great! 

Try to give some small description and link PR to the respective issue. [(like this)](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)

File: src/components/Homepage.jsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
+import React, { useState, useEffect } from "react";
+import "./Homepage.css";
+import Button from "react-bootstrap/Button";
+import axios from "axios";
+import store from "./store";
+import { DATASET_ADD, DATASET_SELECTED } from "./actions";
+import Datasets from "./Dataset";
+
+const Homepage = () => {
+  // objects for function function to run
```
Comment (RoneelV) [inline src/components/Homepage.jsx]: typo?

File: src/components/Homepage.jsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  const [showHoverText, setShowHoverText] = useState(false);
+  const [showParameter, setShowParameter] = useState(false);
+  //---------------------------//-----------------------------------------//
+
+  //---------------------------//-----------------------------------------//
+  // gets data from Dataset.jsx and adds to state
+  useEffect(() => {
+    // console.log(Datasets);
```
Comment (RoneelV) [inline src/components/Homepage.jsx]: Not necessary, but probably removing commented console logs is better idea.

File: src/components/Homepage.css [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  background-size: cover;
+  min-height: 700px;
+}
+
+.mainQuestion {
+  padding-top: 30px;
+  margin-bottom: 50px;
+}
+
+.button {
```
Comment (RoneelV) [inline src/components/Homepage.css]: Can be dangerous, give better class name and follow a convention. Suggestion: [BEM](http://getbem.com/)

**Review by RoneelV — COMMENTED** [review]
Have reviewed most parts, LGTM.

**Comment by smit2k14** [comment]
A couple of changes and adding on what Roneel just said:
-  instead of just hovering above to see the dataset, let us keep the options open always and on hover just show some image of the data.
-  Regarding the project structure, it is not advisable to keep different App.js and App.css files - instead I'll lay down the file structure of how stuff is supposed to be. The components folder was a good move, but things further need to be subcategorized down to individual folders for each of the components. Also, homepage is not a component, it is a page, so it would be better to keep that in a separate pages folder. 
- No need for simple comments like "Gets the data", etc. Also, no need for 
`//---------------------------//-----------------------------------------//`
such comments. Let's try and keep the code structuring simple.
- Instead of keeping jsx files, let us keep js files. Also, if possible using typescript would be a definite bonus because over the long run, it would be much more useful. Can pass up on that though.

**Comment by smit2k14** [comment]
give @AkMo3 75 points

**Pull request merged**